elastic/kibana
# [Visualize] Removes the dashboard callout for users without permission

## Summary

Closes #88724. This PR checks the user's dashboard permissions and if they don't have the permission to create a new dashboard it doesn't display the callout.
![image](https://user-images.githubusercontent.com/17003240/106569362-15516b80-653d-11eb-8935-761e2e1e282a.png)

As a bonus, this PR also fixes the types for `VisualizeCapabilities` that until now are typed to `any`.

### How to test it
 - Create a role which doesn't give permissions to the users to create a new Dashboard. 
 - Assign a user to this role
 - Login with the new user
 - Check that the callout is not displayed
